content/docs/command-reference/remove.md

```diff
@@ -8,15 +8,15 @@ Remove DVC-tracked files or directories from the <abbr>workspace</abbr>.
 usage: dvc remove [-h] [-q | -v] [-o | -p] [-f] targets [targets ...]
 
 positional arguments:
-  targets        DVC-files to remove.
+  targets        `.dvc` files to remove.
 ```
 
 ## Description
 
 This command safely removes data files or directories that are tracked by DVC
 from the <abbr>workspace</abbr>. It takes a
 [DVC-File](/doc/user-guide/dvc-file-format) as input, removes all of its outputs
```

Comment:
Missed this one @utkarshsingh99 

Please find the term non-case-sensitive in all the docs of this PR (and throughout, for later PRs).

Reply:
Okay, I think you have already changed this one.

What about line 18 of this file?
`DVC-file`.
I'm guessing I should change it to `.dvc` file.
I'm not sure how grepping DVC-file missed it though.

Reply:
> What about line 18 of this file?

Yes, that's what I'm talking about. It's DVC-File with capital F (which is wrong BTW). Please grep the term non-case-sensitive in all the docs.

Reply:
Also, per https://github.com/iterative/dvc.org/pull/1403#pullrequestreview-426045191 this whole sentence is wrong actually. It should be:

> It takes a stage name (see `-n` option of `dvc run`) or a [`.dvc` file](/doc/user-guide/dvc-file-format) as `target`, removes all of its outputs (`outs` field), and optionally removes the stage entry in [`dvc.yaml`(/doc/user-guide/dvc-file-format) or the `.dvc` file itself.

Cc @efiop is that explanation fully correct? Thanks again.

Reply:
> Please grep the term non-case-sensitive in all the docs

But considering this is just the 2nd chunk, should I restrict the grep search to just the basic concepts folder and the command-reference folder (without the subfolders)?

Reply:
Yes but keep this in mind for all the remaining changes in your original branch — should be double checked at some point.

Reply:
> Cc @efiop is that explanation fully correct? Thanks again.

@jorgeorpinel It is not if we are talking about 1.0: So `dvc remove ...` in 1.0:

It takes a stage name (see -n option of dvc run) or a .dvc file as target, removes the stage entry in [dvc.yaml(/doc/user-guide/dvc-file-format) or the .dvc file itself and optionally removes the outputs.

Reply:
Yes, we're talking about 1.0.

Ok thanks Ruslan! Moved this to https://github.com/iterative/dvc.org/pull/1408#pullrequestreview-426658412.